conduct.md

```diff
@@ -88,9 +88,11 @@ TODO: Write an introductory paragraph.
 
 #### Acceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Team members should finalize decisions when the team reaches a majority vote.
```

Comment:
The other group that is currently trying to merge to the main repository is saying "Students should ...", rather than "Team members should...". We should probably switch to saying "Students" for consistency.

Reply:
Thanks for the review. I thought about using "students should" as well, but in the code of conduct formatting issue tracker, we said: 

> but if needed use the terms "team members" or "teams" to describe groups of students. I think this is probably the best solution for consistency purposes. Let me know if you disagree!

So, I thought it was more appropriate to use" team members" here because decisions are made between team members rather than all the students in the class.

Reply:
That is a good point and since "team members" **is** defined in the Formatting issue already, this probably does not need changed. Please disregard my previous comment and feel free to resolve this conversation without making any changes.

Reply:
What if "Teams" is used for all of the actions the team should take together (e.g. "Teams should consider and discuss all possible ideas of each student before moving forward.") and "Students" is used for all of the actions students should take independently (e.g. "Students should not take final decisions personally."? What do you think @Ochirsaikhan and @PaigeCD?

Reply:
Okay. Here are my upgraded version of acceptable decisions:

#### Acceptable Conduct

- Teams should finalize decisions when the team reaches a majority vote.
- Teams should consider and discuss all possible ideas of each student before moving forward.
- Teams should document how they reached the final decisions.
- Teams should make space for quieter voices on the team to speak.
- Students should not take final decisions personally.

What do you think?

Reply:
@Ochirsaikhan, these points look great!

Reply:
@Ochirsaikhan I agree, this looks good! Please commit these changes